apps/dashboard/src/lib/components/Course/components/People/InvitationModal.svelte

```diff
@@ -68,34 +125,28 @@
         name: tutor.text,
         orgName: $currentOrg.name,
         orgSiteName: $currentOrg.siteName,
-        courseName: $course.title
+        courseName: $course?.title
       });
     }
 
-    addGroupMember(membersStack).then(async (membersAdded) => {
-      if (!$course?.group?.id) return;
+    await addGroupMember(membersStack);
 
-      const group = await fetchGroup($course?.group?.id);
```

Comment:
@tunjiadeyemi is there a reason you removed this fetching?

Reply:
i was referencing the invitation modal for testpacks but this has been fixed in the latest push